deploy/crds/operator.ibm.com_v1alpha1_mongodb_cr.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operator.ibm.com/v1alpha1
+kind: MongoDB
+metadata:
+  name: example-mongodb
+spec:
+  replicas: 3
+  pullSecret: image-pull-secret
+  imageRegistry: hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-amd64
```

Comment:
can you push the images of three archs to quay.io/opencloudio and build multiarch of them?

make the pullSecret optional or remove it.